benchmarks/perception/a3_stereo_image_proc/launch/trace_a3_stereo_image_proc.launch.py

```diff
@@ -0,0 +1,93 @@
+import os
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+from tracetools_launch.action import Trace
+from tracetools_trace.tools.names import DEFAULT_EVENTS_ROS
+from tracetools_trace.tools.names import DEFAULT_EVENTS_KERNEL
+from tracetools_trace.tools.names import DEFAULT_CONTEXT
+ 
+def generate_launch_description():
+     # Trace
+    trace = Trace(
+        session_name="a3_stereo_image_proc",
+        events_ust=[
+            "robotperf_benchmarks:*",
+            "ros2_image_pipeline:*",
+            "ros2:*"
+            # "lttng_ust_cyg_profile*",
+            # "lttng_ust_statedump*",
+            # "liblttng-ust-libc-wrapper",
+        ]
+        + DEFAULT_EVENTS_ROS,
+        context_fields={
+                'kernel': [],
+                'userspace': ['vpid', 'vtid', 'procname'],
+        },
+        # events_kernel=DEFAULT_EVENTS_KERNEL,
+        # context_names=DEFAULT_CONTEXT,
```

Comment:
Why not remove the commented lines?

Reply:
+1 to this. My fault probably for leaving it in other examples.